components/browser/state/src/main/java/mozilla/components/browser/state/action/BrowserAction.kt

```diff
@@ -320,6 +320,12 @@ sealed class WebExtensionAction : BrowserAction() {
     data class UpdateWebExtensionEnabledAction(val extensionId: String, val enabled: Boolean) :
         WebExtensionAction()
 
+    /**
+     * Updates the [WebExtensionState.enabled] flag.
+     */
+    data class UpdateWebExtensionAllowedInPrivateBrowsingAction(val extensionId: String, val allowed: Boolean) :
```

Comment:
There's a test missing in `WebExtensionActionTest` for the new action. It's a quick one though :).